wifiphisher/common/constants.py

```diff
@@ -85,6 +85,38 @@
 GR = '\033[37m'  # gray
 T = '\033[93m'   # tan
 
+# Logging configurations
+# possible values for debug levels are:
+# CRITICAL, ERROR, WARNING, INFO, DEBUG, NOTSET
+DEBUG_LEVEL = 'INFO'
+LOG_FILEPATH = 'wifiphisher.log'
```

Comment:
Let users specify the `debug level` and `filepath` in `constant.py`

Reply:
nice 👍

Reply:
Also maybe instead of `DEBUG_LVEVEL` we can use `LOG_LEVEL` as I think it's more clear. What do you think?

Reply:
Sure!